scripts/data_guide/data_guide.qmd

```diff
@@ -431,194 +432,227 @@ for (center in sort(processed_centers)) {
-* Data Description: Interval in days from DOB to date of last contact.
+`SAMPLE_TYPE_DETAILED`
 
-`INT_DOD`
+* Expected Values: Primary tumor, Metastasis site unspecified, Local recurrence, Lymph node metastasis, Not applicable or hematologic malignancy, Distant organ metastasis, Not Collected, Not otherwise specified
+* Data Description: The specimen's detailed sample type based on its location.
 
-* Expected Values: [Integer], <6570, >32485, Not Collected, Not Released, Not Applicable, Unknown
-* Data Description: Interval in days from DOB to DOD.
+`SEQ_ASSAY_ID`
 
-`DEAD`
+* Expected Values: [Center]-[Panel]
+* Data Description: The institutional assay identifier for genomic testing platform. Components are separated by hyphens, with the first component corresponding to the Center's abbreviation. All specimens tested by the same platform should have the same identifier.
 
-* Expected Values: TRUE, FALSE, Not Collected, Not Released, Not Applicable, Unknown
-* Data Description: Is this patient known to be deceased?
+`CANCER_TYPE`
 
-`BIRTH_YEAR` [Not available for public releases]
+* Expected Values: Non-Small Cell Lung Cancer (example)
+    http://oncotree.mskcc.org/#/home?version=`r strsplit(oncotree_link, "=")[[1]][2]`
+* Data Description: The primary cancer diagnosis label, or "main type", based on the OncoTree ontology. For example, the OncoTree code of LUAD maps to: “Non-Small Cell Lung Cancer”.
 
-* Expected Values: [Integer], Unknown, cannotReleaseHIPAA, withheld
-* Data Description: The four-digit year associated with the patient's date of birth.
+`CANCER_TYPE_DETAILED`
+
+* Expected Values: Lung Adenocarcinoma (example)
+    http://oncotree.mskcc.org/#/home?version=`r strsplit(oncotree_link, "=")[[1]][2]`
+* Data Description: The detailed primary cancer diagnosis label based on the OncoTree ontology. For example, the OncoTree code of LUAD maps to the label: "Lung Adenocarcinoma (LUAD)".
 
 `SAMPLE_CLASS` [Not available for public releases]
 
 * Expected Values: Tumor, cfDNA
-* Data Description: Annotate samples as Tumor or cfDNA samples.  cfDNA samples are not available for public releases.
+* Data Description: This sample category of either Tumor or circulating cell-free DNA (cfDNA) type. cfDNA samples are filtered out in public releases.
 
 `SEQ_YEAR` [Not available for public releases]
 
 * Expected Values: [Integer]
-* Data Description: Each record has a year of sequencing.
-
-`AGE_AT_SEQ_REPORT_DAYS` [Not available for public releases]
-
-* Expected Values: [Integer], >32485, <6570, Unknown
-* Data Description: The interval in days between the patient's date of birth and the date of the sequencing report that is associated with the sample.  The interval is masked for >32485 and <6570.
+* Data Description: The year the sample was sequenced.
+
+### Linking clinical data to genomic data
+
++---------------------------+-----------------------------+----------------------+-----------------------+
```

Comment:
I personally use https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify to re-format markdown tables. It shouldn't cause problems not being formatted, but it's a bit nicer when looking at the non-rendered content.